File: js/problem-1012.js
// https://www.beecrowd.com.br/judge/en/problems/view/1012

const GEOMETRIC_SHAPES = ["TRIANGULO","CIRCULO","TRAPEZIO","QUADRADO","RETANGULO"];
let input = require('fs').readFileSync('/dev/stdin', 'utf8'),
[a, b, c] = input.split(' ').map(value => Number(value)),
rightTriangle = (a * c) / 2,
circle = 3.14159 * Math.pow(c, 2),
trapezium = ((a + b) * c) / 2,
square = b * b,
rectangle = a * b,
areas = [rightTriangle, circle, trapezium, square, rectangle];

for(let i = 0; i < areas.length; i++){
    console.log(`${GEOMETRIC_SHAPES[i]}: ${areas[i].toFixed(3)}`);
}

File: python/problem-1012.py
# https://www.beecrowd.com.br/judge/en/problems/view/1012

a, b, c = map(lambda item: float(item), input().split())

right_triangle = (a * c) / 2
circle = 3.14159 * c ** 2
trapezium = ((a + b) * c) / 2
square = b * b
rectangle = a * b

areas = [right_triangle, circle, trapezium, square, rectangle] 
geometric_shapes = ["TRIANGULO", "CIRCULO", "TRAPEZIO", "QUADRADO", "RETANGULO"]


for geometric_shape, area in zip(geometric_shapes, areas):
    print(f"{geometric_shape}: {area:.3f}")